doc/manual.asciidoc

```diff
@@ -204,7 +204,9 @@ Several placeholders are available:
 `%o`:: Overall rate of finished edges per second
 `%c`:: Current rate of finished edges per second (average over builds
 specified by `-j` or its default)
-`%e`:: Elapsed time in seconds.  _(Available since Ninja 1.2.)_
+`%e`:: Elapsed time in hh:mm:ss format.  _(Available since Ninja 1.2.)_
+`%E`:: Remaining time (ETA) in hh:mm:ss format.  _(Available since Ninja 1.12.0.)_
+`%P`:: The percentage of time elapsed out of predicted total runtime.  _(Available since Ninja 1.12.0.)_
```

Comment:
A note that this includes the `%` would be handy. The padded-to-width-3 may also be relevant.